### PR TITLE
Feature/verifypage

### PR DESCRIPTION
**Pull request by Sagargajare**
1. Verify Page Added
2. Linting Setup with prettier
3. Login page Added
4. Create Page Added 
5. All UI Static Added

### CODE REVIEW

**Comment by AffableSarthak** [comment]
Please fix the build errors and push again @Sagargajare

File: components/dashboard/createComp/ContentType/index.tsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+      title: 'Video + Article ',
+      icon: BsEmojiSmile,
+      des: 'go all out record and write',
+      bg: '#8133FF',
+    },
+  ];
+  const [tabIndex, setTabIndex] = React.useState(0);
+
+  const handleTabsChange = (index: any) => {
+    console.log(index);
```
Comment (AffableSarthak) [inline components/dashboard/createComp/ContentType/index.tsx]: Remove all console logs.

File: components/login/index.tsx [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+  Flex,
+  Box,
+  Stack,
+  Button,
+  Text,
+  HStack,
+  Image,
+  Hide,
+} from '@chakra-ui/react';
+// import Image from 'next/image';
```
Comment (AffableSarthak) [inline components/login/index.tsx]: Remove the code that is not used.

File: components/login/index.tsx [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+import useLoadingStore from '../../stores/loadingStore';
+
+export default function LoginCard() {
+  const isLoading = useLoadingStore((state) => state.isLoading);
+  const toggleLoading = useLoadingStore((state) => state.toggleLoading);
+
+  const login = useAuthenticationStore((state) => state.login);
+
+  if (isLoading) {
+    return <div>...LOADING</div>;
```
Comment (AffableSarthak) [inline components/login/index.tsx]: Have a "loading" component. If you do then update this.

File: layouts/CreateLayout.tsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import { Box } from '@chakra-ui/react';
+import React from 'react';
+interface IProps {
+  children: React.ReactNode;
+}
+const CreateLayout = ({ children }: IProps) => {
```
Comment (AffableSarthak) [inline layouts/CreateLayout.tsx:6]: Spacing between blocks of code is required. 
Space between imports, types, and the component itself.

File: api/authentication.ts [hunk 5]
```diff
@@ -19,27 +19,21 @@ export type AuthenticationFailure = {
...
-  CREATOR = "creator",
+  ADMIN = 'admin',
+  CREATOR = 'creator',
 }
 export interface AuthData {
   role: Roles | undefined;
   accessToken: string | undefined;
   refreshToken: string | undefined;
 }
+export type roles = 'Admin' | 'Creator' | undefined;
```
Comment (AffableSarthak) [inline api/authentication.ts:30]: Remove unused types.

File: layouts/DashboardLayout.tsx [hunk 6]
```diff
@@ -16,44 +16,52 @@ import {
...
+  BsBoxArrowDownLeft,
+} from 'react-icons/bs';
+import { IoIosHome } from 'react-icons/io';
+import { IconType } from 'react-icons';
+import { IoIosStats, IoIosBuild, IoIosRocket } from 'react-icons/io';
+import { ReactText } from 'react';
+import TwitterCard from '../components/Sidebar/TwitterCard';
+import useAuthenticationStore from '../stores/authenticationStore';
+import { AuthData } from '../api/authentication';
+// import RightSideBar from "../components/dashboard/RightSideBar";
```
Comment (AffableSarthak) [inline layouts/DashboardLayout.tsx:36]: Same as previous comment

File: components/dashboard/createComp/ContentType/index.tsx (hunk 1 above)
Comment (Sagargajare) [reply components/dashboard/createComp/ContentType/index.tsx]: okay

File: components/login/index.tsx (hunk 2 above)
Comment (Sagargajare) [reply components/login/index.tsx]: okay

**Pull request merged**